src/components/TrancheLine.tsx

```diff
@@ -37,13 +37,33 @@ export const TrancheLine: FunctionComponent<TrancheLineProps> = ({
 		<>
 			<div className={`${getTextClassNames()} flex items-baseline`}>
 				<div
-					style={{ backgroundColor: value && !isDeEmphesised() ? color : "transparent" }}
+					style={{
+						backgroundColor: percentageOfInterest && !isDeEmphesised() ? color : "transparent",
+					}}
 					className="w-2 h-2 rounded-full mr-2"
 				></div>
 				<span>{name}</span>
 			</div>
-			<div className={getTextClassNames()}>{apr ? ratioFormatter.format(apr.toNumber()) : "/"}</div>
-			<div className={getTextClassNames()}>{value ? ratioFormatter.format(value) : "/"}</div>
+			<div className={getTextClassNames()}>
+				{percentageOfPrincipal
+					? ratioFormatter.format(
+							Big(percentageOfPrincipal.toString().replace(",", "")).div(100).toNumber()
+					  )
+					: "/"}
+			</div>
+			<div className={getTextClassNames()}>
+				{percentageOfInterest
+					? ratioFormatter.format(
+							Big(percentageOfInterest.toString().replace(",", "")).div(100).toNumber()
+					  )
+					: "/"}
+			</div>
+			<div className={getTextClassNames()}>
+				{" "}
+				{apr
+					? ratioFormatter.format(Big(apr.toString().replace(",", "")).div(100).toNumber())
```

Comment:
not sure if this always works? isn't this dependent on locale? also, can we abstract this as we are doing the same thing multiple times.